libs/spice/patches/030-include-generated-code.patch

```diff
@@ -0,0 +1,22 @@
+--- a/subprojects/spice-common/common/meson.build
++++ b/subprojects/spice-common/common/meson.build
```

Comment:
Please add patch like it's exported with git format-patch, with subject title and message body.  Like they are to be upstreamed (or serve as a indication even if they dont) and reviewed by others.

Reply:
Done.